docs/index.rst

```diff
@@ -5,28 +5,18 @@
 
 :tocdepth: 3
 
-.. the "raw" directive below is used to hide the title in favor of just the logo being visible
-.. raw:: html
+#################################################
+astropy: A Community Python Library for Astronomy
+#################################################
 
-    <style media="screen" type="text/css">
-      h1 { display:none; }
-    </style>
+**Version**: |version|
```

Comment:
@saimn , why you put version here? RTD gives you the version already, no?

Reply:
we have it currently on the top left, and I think it's useful to have it visible somewhere to avoid confusion, so I copied Numpy's way to show it.